packages/slate/src/changes/at-range.js

```diff
@@ -557,7 +557,12 @@ Changes.deleteForwardAtRange = (change, range, n = 1, options = {}) => {
 
   // If the closest is not void, but empty, remove it
   if (block && !block.isVoid && block.isEmpty && document.nodes.size !== 1) {
+    const nextBlock = document.getNextBlock(block.key)
     change.removeNodeByKey(block.key, { normalize })
+    if (nextBlock && nextBlock.key) {
+      change.moveToRangeOf(nextBlock)
+      change.moveOffsetsTo(0, 0)
```

Comment:
I think this can be summarized as `moveToStartOf(nextBlock)`

Reply:
Yup. Pushed, thanks!